msdk-io/msdk-io-mzml2/src/main/java/io/github/msdk/io/mzml2/data/MzMLBinaryDataInfo.java

```diff
@@ -17,7 +17,9 @@
 
   public static enum MzMLCompressionType {
     NUMPRESS_LINPRED("MS:1002312"), NUMPRESS_POSINT("MS:1002313"), ZLIB(
-        "MS:1000574"), NUMPRESS_SHLOGF("MS:1002314"), NO_COMPRESSION("MS:1000576");
+        "MS:1000574"), NUMPRESS_SHLOGF("MS:1002314"), NO_COMPRESSION(
```

Comment:
These enum constants can be formatted more nicely if you add comments to them, e.g.
NUMPRESS_SHLOGF("MS:1002314"), // Numpress compression, type shlogf, which means ...
NO_COMPRESSION("MS:1000576"), // No compression
NUMPRESS_LINPRED_ZLIB("MS:1002746"), // Numpress compression, type linpred + ZLIB
NUMPRESS_POSINT_ZLIB("MS:1002747"), // Numpress compression, type posint + ZLIB
NUMPRESS_SHLOGF_ZLIB("MS:1002748"); // Numpress compression, type shlogf + ZLIB